extensions/mssql/package.json

```diff
@@ -312,7 +312,7 @@
             "verbose"
           ],
           "default": "off",
-          "description": "%mssql.tracing.desc%"
+          "description": "%mssql.tracingServer%"
```

Comment:
Is this actually needed? Generally we should avoid updating string IDs unless it's important since that resets all the loc string mappings (unless you feel like going in and changing all those too...)